include/z64actor.h

```diff
@@ -357,4 +357,101 @@ typedef enum {
     DOORLOCK_NORMAL_SPIRIT
 } DoorLockType;
 
+typedef enum {
+    NAVI_ENEMY_DEFAULT,
+    NAVI_ENEMY_GOHMA,
+    NAVI_ENEMY_ENGOMA_EGG,
+    NAVI_ENEMY_ENGOMA,
+    NAVI_ENEMY_SKULLTULA_NORMAL,
+    NAVI_ENEMY_SKULLTULA_BIG,
+    NAVI_ENEMY_TAILPASARAN,
+    NAVI_ENEMY_DEKU_BABA,
+    NAVI_ENEMY_DEKU_BABA_BIG,
+    NAVI_ENEMY_DEKU_WITHERED,
+    NAVI_ENEMY_DEKU_SCRUB,
+    NAVI_ENEMY_UNUSED11,
+    NAVI_ENEMY_DODONGO,
+    NAVI_ENEMY_DODONGO2,
```

Comment:
one of these is going to be king dodongo, right

Reply:
Text ID 0x060C is King Dodongo, 0x060D is Dodongo, 0x060E is Baby Dodongo